### DISCUSSION TITLE
GFW模式所有站点都正常，唯独就是google的应用商店打不开

### DISCUSSION THREAD

**Discussion by ghost** [Openwrt-Passwall/openwrt-passwall#2354] [General]
都是默认设置，如果白名单模式和全局模式都正常，GFW模式就无法打开页面，但是别的代理站点没问题，代理列表也加上课应用商店的域名，不起作用

**Comment by handsomeji** [comment]
google应用商店用的是：
googleapis.cn
googleapis.com
xn--ngstr-lra8j.com
这几个域名已经默认就在passwall代理规则列表里面

**Comment by ghost** [comment]
我一天的测试情况是，可以GFW模式的插件 除了bypass这个在GFW模式下可以正常访问谷歌商店，其他的都不行，不单单是passwall，我估计是GFW模式下调用的内部dns解析不正常，bypass可能内置的smartdns双服务器，当然这只是猜测，希望能创建一个虚拟机环境，在所有设置都是默认的情况下用GFW做下测试，看看是哪个环节的问题

**Comment by ghost** [comment]
确认是默认版本对解析还是不完善，我自己编译了一个衔接smartdns的那个版本，可以达到我的要求了
